python-frontend/src/main/java/org/sonar/plugins/python/api/tree/ExceptClause.java

```diff
@@ -54,4 +55,7 @@ public interface ExceptClause extends Tree {
   Expression exceptionInstance();
 
   StatementList body();
+
+  @Beta
+  Token colon();
```

Comment:
_minor_ I would move this above the body, just to be more in-line with the flow of the grammar, as the colon precedes the body.